### ISSUE TITLE
Feature request: Backtest entire TV watchlist

### ISSUE THREAD

**Issue by sirouk** [akumidv/tradingview-assistant-chrome-extension#132]
Can you add a function to do a deep/regular backtest across a watchlist?

It should load up the current saved settings for the symbol. Or perhaps the default.

I also wanted to point out that each iteration should reset the Backtest start/end date, as I have seen a flaw in recent code with other extensions that missed this.

I appreciate your work!

**Comment by GitHub1m** [comment]
I strongly agree!

**Comment by akumidv** [comment]
Ok. I'll plan in new version.

**Comment by GitHub1m** [comment]
> Ok. I'll plan in new version.

Hope this will be implemented in the near future!

**Comment by GitHub1m** [comment]
Greetings. Are you still planning to implement this in the new version? Right now, the watchlist backtest feature is no longer working on the "The Optimiser - TradingView Strategies" extension
 (https://chromewebstore.google.com/detail/the-optimiser-tradingview/emcpjechgmpcnjphefjekmdlaljbiegp).

Thank you for your great extension!

**Comment by GitHub1m** [comment]
Greetings. The watchlist backtest feature is on the "The Optimiser - TradingView Strategies" extension you made is now behind a paywall. Are you still planning to implement this in the new version? I really would love this feature. 

(https://chromewebstore.google.com/detail/the-optimiser-tradingview/emcpjechgmpcnjphefjekmdlaljbiegp).

Thank you for your great extension!

**Comment by akumidv** [comment]
Yes, but I still do not finish with refactoring. So it is not the first priority task.

**Comment by GitHub1m** [comment]
> Yes, but I still do not finish with refactoring. So it is not the first priority task.

OK, looking forward to it, thank you!
